### ISSUE TITLE
Failed to load the atom-css-comb package

### ISSUE THREAD

**Issue by fabriziogiordano** [jchouse/csscomb-atom#12]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.191.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: Mac OS X 10.9.5
**Thrown From**: [atom-css-comb](https://github.com/jchouse/csscomb-atom) package, v0.6.4

### Stack Trace

Failed to load the atom-css-comb package

```
At Cannot read property 'prototype' of undefined

TypeError: Cannot read property 'prototype' of undefined
    at __extends (/Users/giordanof/.atom/packages/atom-css-comb/lib/css-comb-view.coffee:4:199)
    at /Users/giordanof/.atom/packages/atom-css-comb/lib/css-comb-view.coffee:9:5
    at Object.<anonymous> (/Users/giordanof/.atom/packages/atom-css-comb/lib/css-comb-view.coffee:157:5)
    at Object.<anonymous> (/Users/giordanof/.atom/packages/atom-css-comb/lib/css-comb-view.coffee:159:4)
    at Module._compile (module.js:452:26)
    at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
    at Module.load (module.js:347:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
```

### Commands

```
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-css-comb, v0.6.4
autocomplete-plus, v2.10.0
linter, v0.12.0

# Dev
No dev packages
```

**Comment by fabriziogiordano** [comment]
by the way @jchouse you are great! Sorry to bombard with issue.
Thank you for this great package.

**Comment by jchouse** [comment]
All OK! My Mac broke and since Windows can't update the package. As soon as I can, I'll fix it. A little bear.

**Comment by jchouse** [comment]
Fix in 1.0.1
